Sprint #19.adoc

```diff
@@ -29,9 +29,45 @@
 * TASK: make hasAttachment searchable via mailbox-api
 * TASK: make GetMessagesList call the search api
 
+Remark. If you want to call here the search API for Messages you will have to problems :
+ - It returns a list of UID. You may need to add information about mailboxes depending on your implementations choices
```

Comment:
You mean we have mail uid but not the mailbox ? It's pointless, an uid is related to a mailbox ...